src/Common/src/CoreLib/System/IO/Path.cs

```diff
@@ -910,29 +910,21 @@ private static string GetRelativePath(string relativeTo, string path, StringComp
         /// <summary>
         /// Trims one trailing directory separator beyond the root of the path.
         /// </summary>
-        public static string TrimEndingDirectorySeparator(string path) =>
-            EndsInDirectorySeparator(path) && !PathInternal.IsRoot(path.AsSpan()) ?
-                path.Substring(0, path.Length - 1) :
-                path;
+        public static string TrimEndingDirectorySeparator(string path) => PathInternal.TrimEndingDirectorySeparator(path);
```

Comment:
This is changing some files that are compiled into Corelib as well.  Have you validated that Corelib successfully builds if you copy all of these Common/src/CoreLib files over to your coreclr repo?

Reply:
@stephentoub I copied the changes for `Path.cs` and `PathInternal.cs` to coreclr and it finished building successfully:

![image](https://user-images.githubusercontent.com/1175054/68801035-17659c80-0610-11ea-8c6c-9c6917296d84.png)